partner_industry/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3